submissions/blueprint-rant/blueprint-rant.md

```diff
@@ -0,0 +1,51 @@
+
+![](Green circuit maker screenshot)
+*A simple but large footprint green circuit maker I recently designed to allow the production of 2 belts of green circuits, using only low tier assemblers. Notice the unique and slightly odd design, because why not?.*
+
+By just importing a blueprint every time you need to make something new, you aren't problem solving and designing, you are a construction bot blinding placing racks of machines, and plumbing up the inputs and outputs. That to me really doesn't sound very fun. It sounds like you're playing the fun*ish* in-game scenario, transport belt madness rather than pursuing your own destiny in freeplay.
+
+### Unless, blueprint importing is good?
+
+I will concede, in some **RARE** situations, importing blueprints can be handy to avoid several minutes of monotony. The perfect example of this is solar setups, generally tiling masses containing solar panels and accumulators, in the hope of one day cleanly powering your factory. Spending 5-10 minutes faffing around, trying to perfectly fill up the space around one roboport may sound like fun to some people, but this is a problem that doesn't particularly change depending on the game scenario. Whether you have just researched solar panels or are several hundred hours into maximising the SPM of a save, your solar panel setups are gonna look basically the same, with probably the main difference being the size of the blueprint placed at a single time.
+
+![](Generic tiling Solar Array blueprint)
+
+Balancers, *the black magic creations*, are another example of something that most players want to use, with fair reason, but don't want to learn and understand the maths and mechanics behind them. Figuring out the exact combination of belts, splitters and undergrounds to make the ratios perfect make sounds like a fantastic sunday afternoon for some people, but many would find it dull and unexciting. In both these scenarios, allowing the importing of blueprints will *likely* improve the players enjoyment of the game as they wont get stuck making monotonous blueprints or diving into the maths of balancers.
+
+![](Factorio balancer overview)
+
+### Blueprint library and copy/paste
+
+Once you have built a thing once in a world, its pretty common to need to build it again later, be it miner setups, train stops, train tracks, solar setups, bus layouts et cetera. This is especially compounded in multiplayer games, where each player using their own rail standards can cause chaos and destruction. In these situations, being able to copy, paste and share blueprints using the blueprint library, allows a much smoother and less chaotic experience. I believe the key difference here is, you are recreating soemthing you have already build in the given save, not importing something from somewhere else. As mentioned by *Rseding91* earlier, the ability to use blueprints can also so quickly ruin a multiplayer save, so the tradeoff here is important to think about.
+
```

Comment:
Maybe just pull the section together again with a small paragraph. Saying again something like `so yes, sometimes blueprints are useful, but only in a few cases. Be careful to not use them too often`.